src/ivoc/strfun.cpp

```diff
@@ -21,10 +20,6 @@ extern int nrn_is_artificial(int);
 
 extern int hoc_return_type_code;
 
-inline unsigned long key_to_hash(String& s) {
-    return s.hash();
```

Comment:
`s.hash()` could have been only new thing in `String` I guess 😅

Reply:
The usage is for `std::map` so really useless.